test/library/standard/List/testJson.good

```diff
@@ -58,7 +58,13 @@ Writing list:
 [{A: 1, B: 2}, {Y: 4, X: 3}]
 
 Wrote JSON: 
-[{"A": 1, "B": 2}, {"Y": 4, "X": 3}]
+[{
```

Comment:
The whitespace changes here surprise me, what caused that?

Reply:
The new Json serializer has a different whitespace policy than the old %jt stuff that's built into the IO runtime.  Both are valid JSON though.